api/cpp/include/slint.h

```diff
@@ -222,6 +223,11 @@ class WindowAdapterRc
         cbindgen_private::slint_windowrc_set_physical_size(&inner, &size);
     }
 
+    void dispatch_key_event(const slint::KeyEvent &event)
```

Comment:
On second thought, I'd like to replace this with two overloads that take the press and release event and remove the bare KeyEvent class.